pkg/maps/policymap/policymap_privileged_test.go

```diff
@@ -106,6 +105,5 @@ func (pm *PolicyMapTestSuite) TestDeleteNonexistentKey(c *C) {
 	err := testMap.Map.Delete(&key)
 	c.Assert(err, Not(IsNil))
 	var errno unix.Errno
-	c.Assert(errors.As(err, &errno), Equals, true)
```

Comment:
Why was this assertion removed?

Reply:
I'm uncertain.. My guess is that the change manifests here after rebasing master. 

Currently sorting out issues with swagger: https://github.com/cilium/cilium/pull/10336 Once that's done I'll do another sweep on these changes.